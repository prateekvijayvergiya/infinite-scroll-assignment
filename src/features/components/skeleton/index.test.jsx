import Component from './'
import makeSnapshot from './../../testing/makeSnapshot'

jest.mock('@material-ui/lab/Skeleton', () => 'Skeleton')
jest.mock('@material-ui/core/Card', () => 'Card')
jest.mock('@material-ui/core/CardActionArea', () => 'CardActionArea')
jest.mock('@material-ui/core/CardContent', () => 'CardContent')

describe('LoadingSkeleton', () => {
  it('renders correctly', () => {
    makeSnapshot(<Component />)
  })
})
