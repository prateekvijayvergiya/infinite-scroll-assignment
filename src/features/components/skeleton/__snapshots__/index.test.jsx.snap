// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSkeleton renders correctly 1`] = `
<Card
  className="makeStyles-root"
>
  <CardActionArea>
    <CardContent>
      <Skeleton
        height={200}
        variant="rect"
        width={170}
      />
      <Skeleton
        height={50}
        variant="text"
        width={170}
      />
    </CardContent>
  </CardActionArea>
</Card>
`;
